hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/dispatcher/ReadableStoreFactory.java

```diff
@@ -62,6 +68,7 @@ ReadableScheduleStore.class, new StoreEntry(ScheduleService.NAME, ReadableSchedu
      *
      * @param state the {@link HederaState} to use
      */
+    @Inject
```

Comment:
A `ReadableStoreFactory` should not be injected.

Reply:
Not sure how else to provide to `MonoHandleContext`. Do you think it should be provided in `createReadableStore` ?

Reply:
Added Dagger injected State and Provider<StoreFactory> , so it will get latest mutable state at start of `onHandleConsensusRound` , and will create new factory for each handle. This is added as temporary solution until `HandleContextImpl` is implemented.